projects/netgrif-components-core/src/lib/data-fields/base-component/abstract-base-data-field.component.ts

```diff
@@ -13,8 +14,10 @@ export abstract class AbstractBaseDataFieldComponent<T extends DataField<unknown
     @Input() public dataField: T;
     @Input() public formControlRef: FormControl;
     @Input() public showLargeLayout: WrappedBoolean;
+    public _saveDataInform: boolean;
 
-    constructor(@Optional() @Inject(DATA_FIELD_PORTAL_DATA) dataFieldPortalData: DataFieldPortalData<T>) {
+    constructor(@Optional() @Inject(DATA_FIELD_PORTAL_DATA) dataFieldPortalData: DataFieldPortalData<T>,
+                @Optional() @Inject(NAE_SAVE_DATA_INFORM) _saveDataInform: boolean | null = false) {
```

Comment:
as far as I know, setting default value for an `@Optional` injector does not work as desired.
`null` is a falsy value, so it still works as intended.

Reply:
Yes, if there is no such injection token provided, it will be considered as null, which will indicate falsy value, but I think, the default value is used because of without that it would be needed to add the `@Optional() @Inject(NAE_SAVE_DATA_INFORM) _saveDataInform: boolean` to constructors of every subclass of `AbstractBaseDataFieldComponent`